src/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
```

Comment:
```suggestion
  <React.StrictMode>
```
Un dato curioso https://es.reactjs.org/docs/strict-mode.html